libs/expo/betterangels/src/lib/screens/Note/NoteTitle.tsx

```diff
@@ -15,7 +15,8 @@ export default function NoteTitle({
         {note?.purpose}
       </TextBold>
       <TextRegular size="sm" mb="sm">
-        {format(new Date(note?.interactedAt), 'MM/dd/yyyy')}
+        {!!note?.interactedAt &&
```

Comment:
`interacted_at` can't be null. If this check aligns with convention/best practice, I'm all for it. Just making sure it makes sense

Reply:
Same comment as above for `dateOfBirth`